integration-tests/test.Dockerfile

```diff
@@ -13,6 +13,7 @@ FROM ${BASE_IMAGE}:${IMAGE_VERSION}
 RUN mkdir -p /go/testdir/integration-tests/scripts
 COPY --from=build-env /go/pkg /go/pkg
 COPY --from=build-env /go/testdir/integration-tests/*.test /go/testdir/integration-tests/
+COPY --from=build-env /go/testdir/integration-tests/ccip-tests/*.test /go/testdir/integration-tests/
```

Comment:
needed?

Reply:
Yes, this actually pulls the tests that are built in CCIP into the runner image. Without it we just have core load tests (or none).